packages/react-table/src/demos/Table.md

```diff
@@ -243,8 +260,185 @@ class BulkSelectTableDemo extends React.Component {
+          cells: ['child - 1']
+        },
+        {
+          isOpen: false,
+          cells: ['parent - 2', 'two', 'three', 'four', 'five']
+        },
+        {
+          parent: 3,
+          cells: ['child - 2']
+        },
+        {
+          isOpen: false,
+          cells: ['parent - 3', 'two', 'three', 'four', 'five']
+        },
+        {
+          parent: 5,
+          cells: ['child - 3']
+        }
+      ]
+    };
+    this.onCollapse = this.onCollapse.bind(this);
+    this.toggleCollapseAll = this.toggleCollapseAll.bind(this);
+    this.onSelect = this.onSelect.bind(this);
+    this.toggleSelectAll = this.toggleSelectAll.bind(this);
+  }
+
+  onCollapse(event, rowKey, isOpen) {
+    const { rows, expandedRows, expandCollapseToggle } = this.state;
+    const expandableRowLength = this.state.rows.filter(row => row.isOpen !== undefined).length;
+    /**
+     * Please do not use rowKey as row index for more complex tables.
+     * Rather use some kind of identifier like ID passed with each row.
+     */
+    rows[rowKey].isOpen = isOpen;
+    const updatedExpandedRows = isOpen ? expandedRows + 1 : expandedRows - 1;
+    this.setState({
+      rows,
+      expandedRows: updatedExpandedRows,
+      expandCollapseToggle:
+        updatedExpandedRows === expandableRowLength
+          ? 'collapse'
+          : updatedExpandedRows === 0
+          ? 'expand'
+          : expandCollapseToggle
+    });
+  }
+
+  toggleCollapseAll(collapse) {
+    const expandableRowLength = this.state.rows.filter(row => row.isOpen !== undefined).length;
+    const updatedRows = this.state.rows.map(row =>
+      row.isOpen !== undefined
+        ? {
+            ...row,
+            isOpen: !collapse
+          }
+        : row
+    );
+    this.setState({
+      rows: updatedRows,
+      expandedRows: collapse ? 0 : expandableRowLength,
+      expandCollapseToggle: collapse === false ? 'collapse' : 'expand'
+    });
+  }
+
+  onSelect(event, isSelected, rowId) {
+    let isChecked = null;
+    let selectedRows = this.state.selectedRows;
+    const selectableRowLength = this.state.rows.filter(row => row.parent === undefined).length;
+    let rows;
+    if (rowId === -1) {
+      rows = this.state.rows.map(row => {
+        row.selected = isSelected;
+        return row;
+      });
+      isChecked = isSelected;
+      selectedRows = isSelected ? selectableRowLength : 0;
+    } else {
+      rows = [...this.state.rows];
+      rows[rowId].selected = isSelected;
+      selectedRows = isSelected ? selectedRows + 1 : selectedRows - 1;
+      isChecked = selectedRows === 0 ? false : selectedRows === selectableRowLength ? true : null;
+    }
+    this.setState({
+      rows,
+      isChecked,
+      selectedRows
+    });
+  }
+
+  toggleSelectAll(checked) {
+    this.onSelect(null, checked, -1);
+  }
+
+  render() {
+    const { columns, rows, expandCollapseToggle, isChecked } = this.state;
+
+    return (
+      <React.Fragment>
+        <Toolbar>
```

Comment:
If you add `<ToolbarContent>` and use a `<ToolbarGroup>`, the alignment is improved. 

<img width="850" alt="Screen Shot 2020-10-15 at 11 03 18 AM" src="https://user-images.githubusercontent.com/35148959/96155957-25820400-0ed6-11eb-95bf-3107e13560e4.png">

@mcarrano wdyt, should it look like this?